src/node.cc

```diff
@@ -2286,13 +2286,7 @@ static void DLOpen(const FunctionCallbackInfo<Value>& args) {
   }
 
   if (mp->nm_version == -1) {
-    if (env->EmitNapiWarning()) {
-      if (ProcessEmitWarning(env, "N-API is an experimental feature and could "
-                                  "change at any time.").IsNothing()) {
-        dlib.Close();
-        return;
-      }
-    }
+    // N-API module this is ok
```

Comment:
I think it would be better to combine the `mp->nm_version == -1` check with the one in the `else if`